lib/api/apiUtils/authorization/prepareRequestContexts.js

```diff
@@ -56,12 +62,18 @@ function prepareRequestContexts(apiMethod, request, sourceBucket,
             objectGetAction, 's3');
         const putRequestContext = generateRequestContext('objectPut');
         requestContexts.push(getRequestContext, putRequestContext);
-        if (apiMethodAfterVersionCheck === 'objectCopy' &&
-        request.headers['x-amz-tagging'] &&
-        request.headers['x-amz-tagging-directive'] === 'REPLACE') {
-            const putTaggingRequestContext =
-                generateRequestContext('objectPutTagging');
-            requestContexts.push(putTaggingRequestContext);
+        if (apiMethodAfterVersionCheck === 'objectCopy') {
```

Comment:
As discussed, apiMethodAfterVersionCheck for 'objectCopy' is 'objectCopy' so covered for both.